### PR TITLE
Don't install dependencies into tool virtualenv

### PR DESCRIPTION
**Pull request by DRMacIver**
We have a bunch of dependencies in tools.txt that are only there so that isort doesn't sort them incorrectly.

It turns out that we can just tell isort how to treat those imports explicitly. This updates the build to do that so we don't get pyup pull requests for tools.txt for them (though we'll still get them for the other files they're in, it just means the tools virtualenv will change less often).

### CODE REVIEW

File: .isort.cfg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[settings]
+
```
Comment (alexwlchan) [inline .isort.cfg]: Do we need the blank line?

File: requirements/tools.txt [hunk 2]
```diff
@@ -5,18 +5,14 @@ autoflake==0.7
...
 docutils==0.13.1
-Faker==0.7.11
 flake8==3.3.0
-flaky==3.3.0
 imagesize==0.7.1
 isort==4.2.5
 Jinja2==2.9.6
 MarkupSafe==1.0
 mccabe==0.6.1
-numpy==1.12.1
```
Comment (alexwlchan) [inline requirements/tools.txt]: Are these really the only packages we can get rid of? Not even `pytz`?

File: .isort.cfg (hunk 1 above)
Comment (DRMacIver) [inline .isort.cfg]: Probably not? But I think it looks better to separate the header from the contents by a blank line.

File: requirements/tools.txt (hunk 2 above)
Comment (DRMacIver) [inline requirements/tools.txt]: Not sure. I didn't check in detail. I think pytz is there as a sphinx dependency as well as on its own merits.

The main ones I wanted to get rid of were numpy and Django, which are relatively slow to install.

**Comment by Zac-HD** [comment]
Closed by #623

**Pull request closed without merging**